solo5-bindings-genode.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix=${pcfiledir}/../..
+exec_prefix=${prefix}
+includedir=${prefix}/include/solo5-bindings-genode
+libdir=${exec_prefix}/lib/solo5-bindings-genode
+
+Name: solo5-bindings-genode
+Version: 0.4.0
+Description: Solo5 sandboxed execution environment (Genode target)
+Cflags: !CFLAGS! -isystem ${includedir}/crt -I${includedir}/solo5
+Libs: -nostdlib -z max-page-size=0x1000 -gc-sections -Ttext=0x01000000 --dynamic-linker=ld.lib.so --eh-frame-hdr -rpath-link=. -T ${libdir}/genode_dyn.ld ${libdir}/solo5.lib.so
```

Comment:
This should just be `!GENODE_LDFLAGS!`, shouldn't it? Or is there a 2nd set of Genode LDFLAGS involved here, different from that defined in the Solo5 Makefiles?